app/controllers/catalog_controller.rb

```diff
@@ -118,6 +119,7 @@ def index
     # config.show.thumbnail_field = 'thumbnail_path_ss'
 
     config.add_results_collection_tool(:sort_widget)
+    config.add_results_collection_tool(:algorithm_select)
```

Comment:
Can we add this to the module somehow?

Reply:
This is possible:
```

  def blacklight_config
    config = super
    config.add_results_collection_tool(:algorithm_select)
    config
  end
```

but it's not going to work out well. The location of the dropdown will vary depending on the app as each app will have different widgets enabled.